### ISSUE TITLE
Do interviews on initial sound design for "Waves" screen

### ISSUE THREAD

**Issue by jbphet** [phetsims/greenhouse-effect#102]
In a recent design meeting we decided that the soundscape for the "Waves" screen (see #36) has reached the point where it is ready for some initial interviews.  @arouinfar said she would set up and perform the interviews.  I've published a dev version with the latest version of the soundscape, and it should be good to go for this purpose, see https://phet-dev.colorado.edu/html/greenhouse-effect/1.1.0-dev.3/phet/greenhouse-effect_en_phet.html.

Assigning to @arouinfar for the next steps.

**Comment by emily-phet** [comment]
@jbphet sent @Ashton-Morris and I the latest link to discuss today in his absence: https://phet-dev.colorado.edu/html/greenhouse-effect/1.1.0-dev.4/phet/greenhouse-effect_en_phet.html. 

Ashton and I think this is really good, and in a great place for interviews. The discrete sound heard in photon view when there is absorption/re-emission in the atmosphere we think is the least easily interpretable at the default slider position (slider thumb in the middle of the slider). Exploring higher (more photons, more sounds to match up with the re-emission events) or lower slider locations (less photons, easier to connect the few photon re-emissions with sounds) are easier to interpret, we thought this was fine (we didn't interpret it as confusing at the middle slider range, just the location least intuitive to interpret).

**Comment by jbphet** [comment]
Since the sound design for the "Photons" and "Layer Model" screens is now implemented, there isn't much point in interviewing on just the wave screen.  We've decided to close this issue and first do an internal review of the complete sound design (see https://github.com/phetsims/greenhouse-effect/issues/222), which will be followed up by a review by the full design team.  After that, we will likely do interviews that include the sound design as part of full user testing.
